murdock.py

```diff
@@ -308,6 +308,8 @@ def job_hook(s, arg, job):
                 }
         if target_url:
             status["target_url"] = target_url
+        else:
+            status["target_url"] = config.http_root
```

Comment:
Remark: it would probably be cleaner to have this point to `os.path.join(config.http_root, s.base_full_name)`, but that's not were the (shiny) PR overview was put.